### ISSUE TITLE
Unable to Register a Passkey Using a PIN Code

### ISSUE THREAD

**Issue by Yoichiro-Y** [MasterKale/SimpleWebAuthn#501]
## Describe the issue
Environment: Devices that cannot use biometric authentication  (macOS Ventura 13.2.1 clamshell mode and Android emulator virtual terminal Pixel 7 Pro API 33)
Click and press the "Register" button in the [SimpleWebAuthn Example Project](https://github.com/MasterKale/SimpleWebAuthn/tree/master/example)  and try to register passkey, the following error response is returned.
` "error": "User verification required, but user could not be verified"`
The settings for the authenticator are set as follows
```
authenticatorSelection: {
    authenticatorAttachment: "platform",
    residentKey: "preferred",
    userVerification: "preferred"
}
```
In addition to this, I tried several other patterns with different settings for userVerification and residentKey, and the same error occurred.

## Expected behavior
- I expected to be asked for a PIN code or password to unlock the screen lock, and after entering it, I would register a passkey

## Dependencies

- macOS Ventura 13.2.1 
- Chrome 120

### SimpleWebAuthn Libraries

```sh
$ npm list --depth=0 | grep @simplewebauthn
├── @simplewebauthn/browser@_._._
├── @simplewebauthn/server@_._._
# ...
```

**Comment by MasterKale** [comment]
Hello @Yoichiro-Y, thank you for creating an issue about this. Fortunately this is very similar to https://github.com/MasterKale/SimpleWebAuthn/issues/477, which I fixed almost an hour ago in #500. Can you pull down the latest **master** branch and try again? I tested myself on my own MacBook Pro in clamshell mode and was able to complete both registration and authentication without being prompted for my system password (since biometrics are of course unavailable in this configuration.)

**Comment by Yoichiro-Y** [comment]
@MasterKale , thank you for your reply. I tried it with the latest master branch and it works as expected in clamshell mode!
